### PR TITLE
Support raspberrypi3

### PR DESCRIPTION
**Pull request by chbae**
[GDP-128] GDP-ivi9 on RPi3
[GDP-218] GDP 10 port to RPi3: creation

Signed-off-by: Changhyeok Bae <changhyeok.bae@gmail.com>

### CODE REVIEW

**Comment by chbae** [comment]
This PR should be merged after https://github.com/GENIVI/meta-genivi-dev/pull/5.

**Comment by tom--pollard** [comment]
I have no problems here, however I must note that the meta-genivi-dev submodule refs may change if meta-genivi-dev merges a commit before the raspi3 PR.

**Comment by leon-anavi** [comment]
@chbae well done! @tom--pollard I confirm that I was able to successfully build and boot GDP on Raspberry Pi 3. I recommend to merge the patch.

File: gdp-src-build/conf/templates/raspberrypi3.local.conf [hunk 1]
```diff
@@ -0,0 +1,43 @@
+include templates/local.inc
+
+MACHINE = "raspberrypi3"
+
+GPU_MEM = "128"
+
+CORE_IMAGE_EXTRA_INSTALL += "wayland weston"
```
Comment (gunnarx) [inline gdp-src-build/conf/templates/raspberrypi3.local.conf]: Why is this needed in the pi-specific file?  Is there any difference compared to other hardware platforms?

File: gdp-src-build/conf/templates/raspberrypi3.local.conf [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+PREFERRED_VERSION_mesa = "11.%"
+
+PREFERRED_PROVIDER_virtual/egl = "mesa"
+PREFERRED_PROVIDER_virtual/libgles2 = "mesa"
+PREFERRED_PROVIDER_virtual/libgl = "mesa"
+PREFERRED_PROVIDER_virtual/mesa = "mesa"
+PREFERRED_PROVIDER_jpeg = "jpeg"
+
+#
+# Disk Space Monitoring during the build
```
Comment (gunnarx) [inline gdp-src-build/conf/templates/raspberrypi3.local.conf]: Disk space monitoring?  Surely this is not Raspberry Pi specific?

File: gdp-src-build/conf/templates/raspberrypi3.local.conf [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# of the build. The reason for this is that running completely out of space can corrupt
+# files and damages the build in ways which may not be easily recoverable.
+BB_DISKMON_DIRS = "\
+    STOPTASKS,${TMPDIR},1G,100K \
+    STOPTASKS,${DL_DIR},1G,100K \
+    STOPTASKS,${SSTATE_DIR},1G,100K \
+    ABORT,${TMPDIR},100M,1K \
+    ABORT,${DL_DIR},100M,1K \
+    ABORT,${SSTATE_DIR},100M,1K"
+
```
Comment (gunnarx) [inline gdp-src-build/conf/templates/raspberrypi3.local.conf]: Disk space monitoring? Surely this is not Raspberry Pi specific?

@tom--pollard Please double check everything that can be unified across the platforms and put into the .inc file.

Do we even need this or what is the default if this is not specified in local.conf?
Keep complexity down if possible...

Comment (tom--pollard) [inline gdp-src-build/conf/templates/raspberrypi3.local.conf]: @gunnarx I don't see any harm in extracting this chunk into the .inc file

**Comment by tom--pollard** [comment]
@chbae I have merged the PR to meta-genivi-dev, could you amend/rebase this PR so that the submodule commit matches? Thanks

**Comment by chbae** [comment]
@tom--pollard Sure. I'll change it soon.

@gunnarx I'll check your comment in next commit.

Thanks for your comments.

**Comment by chbae** [comment]
@tom--pollard I update meta-genivi-dev.

File: gdp-src-build/conf/templates/raspberrypi3.local.conf (hunk 3 above)
Comment (slawr) [inline gdp-src-build/conf/templates/raspberrypi3.local.conf]: Is that shared text that bitbake uses for fragment integration as in a patch file? I could be wrong..

**Comment by tom--pollard** [comment]
Merged :)
